common/src/main/java/org/figuramc/figura/animation/Keyframe.java

```diff
@@ -83,26 +96,43 @@ private float parseStringData(String data, float delta) {
         return FiguraMod.popReturnProfiler(0f);
     }
 
+    @LuaWhitelist
+    @LuaMethodDoc("keyframe.ph")
     public float getTime() {
         return time;
     }
 
+
     public Interpolation getInterpolation() {
         return interpolation;
     }
 
+    @LuaWhitelist
+    @LuaMethodDoc("keyframe.ph")
+    public String getInterpolate() {
```

Comment:
Also, forgot about this, but `getInterpolation` (verb noun) not `getInterpolate` (verb verb)

Reply:
so uhhhh, that is different from the existing getInterpolation and idk how to get around that
waybe we should move to dev-chat

Reply:
having the same name is fine, maybe add a Void argument? or no, LuaValue arg? don't know which, but
i am physically unable to move to dev-chat